conf/test/test.config

```diff
@@ -58,6 +58,15 @@ params {
 }
 
 process {
+    withLabel: 'sentieon' {
+        ext.sentieon_auth_mech_base64 = secrets.SENTIEON_AUTH_MECH_BASE64
+        ext.sentieon_auth_data_base64 = secrets.SENTIEON_AUTH_DATA_BASE64
+    }
+
+    withName: 'SENTIEON_BWAMEM' {
+        ext.args = "-R \"@RG\\tID:sample_lane\\tPU:lane\\tSM:patient_sample\\tLB:sample\\tDS:fasta\\tPL:seqplatform\""
+    }
```

Comment:
Can e get that properly set up in `conf/modules/aligner.config`?

Reply:
> Can e get that properly set up in `conf/modules/aligner.config`?

Good idea. I'll give it a shot.